pxr/usdValidation/usdUtilsValidators/testenv/testUsdUtilsValidators.cpp

```diff
@@ -113,11 +114,48 @@ TestPackageEncapsulationValidator()
     TF_AXIOM(errors.empty());
 }
 
+static void
+TestFileExtensionValidator()
+{
+    UsdValidationRegistry& registry = UsdValidationRegistry::GetInstance();
+
+    // Verify the validator exists
+    const UsdValidationValidator *validator = registry.GetOrLoadValidatorByName(
+            UsdUtilsValidatorNameTokens->fileExtensionValidator);
+
+    TF_AXIOM(validator);
+
+    const UsdStageRefPtr& stage = UsdStage::Open("failFileExtension.usdz");
+    UsdValidationErrorVector errors = validator->Validate(stage);
```

Comment:
if you scope it, then you can make this const, and also the below one on line 147 a const.